order-api/src/main/java/com/example/orderapi/domain/model/EventFailed.java

```diff
@@ -0,0 +1,31 @@
+package com.example.orderapi.domain.model;
+
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Entity
+@Table(name = "event_failed")
+public class EventFailed {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "event_id")
+    private Long id;
+
+    @Column(name = "event_type", nullable = false)
+    private String eventType;
```

Comment:
```suggestion
    private EventType eventType;
```

type 은 enum 으로 관리해도 좋을것같네요 ㅎㅎ

Reply:
넵! 해당 부분 수정 하겠습니다.